### ISSUE TITLE
Failing to connect to box via ssh - connection closed by remote host

### ISSUE THREAD

**Issue by davidwinter** [hashicorp/vagrant#410]
I'm trying to test out vagrant but am having an issue with creating a basic box. Commands I am running are:

```
>: cd Desktop/
>: mkdir vagrant-test
>: cd vagrant-test/
>: vagrant init lucid32
      create  Vagrantfile
>: vagrant up
[default] Importing base box 'lucid32'...
[default] Matching MAC address for NAT networking...
[default] Running any VM customizations...
[default] Clearing any previously set forwarded ports...
[default] Forwarding ports...
[default] -- ssh: 22 => 2222 (adapter 1)
[default] Creating shared folders metadata...
[default] Booting VM...
[default] Waiting for VM to boot. This can take a few minutes.
[default] Failed to connect to VM!
Failed to connect to VM via SSH. Please verify the VM successfully booted
by looking at the VirtualBox GUI.
>: vagrant ssh
ssh_exchange_identification: Connection closed by remote host
```

Vagrant version 0.7.6 and Virtualbox 4.0.10 on Mac OS X 10.6.8

**Comment by mxrguspxrt** [comment]
Having the same issue when vagrant up or ssh with errors (Warning: Remote connection disconnect. Retrying...) and (ssh_exchange_identification: Connection closed by remote host) with Ubuntu 15.10. 

To fix:
1. Open VirtualBox and GUI
2. If server is not started start it manually
3. mv /etc/network/interfaces.d/eth0.cfg /etc/network/interfaces.d/NEW_INTERFACE_NAME.cfg
4. change eth0 to NEW_INTERFACE_NAME in .cfg

You can get NEW_INTERFACE_NAME ifconfig -a

**Comment by NatsuDragnel** [comment]
vagrant destroy
vagrant up
it works!

**Comment by stefek99** [comment]
@NatsuDragnel works for me!

**Comment by sammous** [comment]
You don't need to destroy, I just rebooted the machine:
vagrant halt
vagrant up
vagrant ssh
And it worked for me. I had this issue probably because I messed around with my ssh keys and I guess since the machine was already running it saved previous keys.

**Comment by mattkingston** [comment]
I encountered the same issue and found that I had a settings in ~/.ssh/config that were used for ALL connections. I was behind a proxy and required something like the following for all outbound connections. Completely forgot this would also be applied to local connections e.g;

`Host *
`    ProxyCommand ...
`    ServerAliveInterval 10

This setting was attempting to make all connections run through the proxy - which is incorrect for local connections. Anyway, that brain fart moment was fixed and I was able to SSH into boxes again.
